calibration/FLORTD/CGINS-FLORTD-01218__20220113.csv

```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1218,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1218_Calibration_Files_2022-01-13.zip > BBFL2W-1218.dev
```

Comment:
the source filename extension is capitalized in Vault BBFL2W-1218.DEV

Reply:
doesn't matter - this is ok